src/instructlab/model/evaluate.py

```diff
@@ -552,6 +597,23 @@ def evaluate(
     with contextlib.suppress(ValueError):
         batch_size = int(batch_size)
 
+    if benchmark == Benchmark.LLMAAJ:
+        from instructlab.model.llmaaj import run_llmaaj
+
+        run_llmaaj(ctx,
```

Comment:
Nothing here should be called llmaaj. 

LLMaaJ is just a way of evaluating answers. MT-Bench is also LLMaaJ, but we don't call it that.


I.e., MT-Bench is to LLMaaJ as MMLU is to multiple-choice.